### ISSUE TITLE
Police Oversight form: Test form submissions

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1047]
Basically a full example of:
- Fill out form
- Submit form
- API gets correct data
- S3 gets the correct files
- Data goes to the correct places
- The files are associated with the correct records
- Confirmation number given to user matches all the data in all the places

Probably worth setting up a couple of dummy emails for this so we can have maintain a full path of:
Dev (make sure we have a good way to see the full loop when running locally)
Staging (make sure we have a nice sandbox that sends to email addresses that stay connected to the sandbox)
Prod (make sure we can configure what email address is getting the data and make a switch over when we're ready to go live)
Bonus: PR apps maybe? Mostly helpful when implementing a new control etc.

**Comment by sergiogcx** [comment]
I could not make a pipeline work with yarn, and I do not have much time to continue on it. We may have to re-think this dependency structure, here are my findings:

The goal was to compile both libraries (`usfs-components`, `officer-form-chapters`) either after install or during or build time in `officer-complaint-form`; that would allow us to see the latest code updates for each repo, I tried different methods to accomplish this (ie, `postinstall` hook to trigger the builds, aliases, checked js(x) regexes, added different preset loaders, other yarn stripts) all to no avail.

*Challenges:*
> - Errors on build time: `You may need an appropriate loader to handle this file type.`, even after adding presets: 'env', 'react', 'es2015' I was able to make it build on the component level, but not on chapter level. I tried different combos, no avail. I tried for instaqnce to have a script to `cd node_module/@cityofaustin/repo` and `yarn build` through a yarn script 'buildrepos'. No luck.

> - Raw code + wepback aliases: It would involve removing the build folder and insert aliases for import references in the webpack config, ie:

	```
	resolve: {
	      alias: {
	        "@cityofaustin/usfs-components": path.resolve(__dirname, 'node_modules/@cityofaustin/usfs-components/src/')
	      }
	    }
	```

*Deal-braker:*

> Even if compiled successfully, the FileUploader component cannot resolve where the endpoint is. The problem is that FileUploader will inherit the API endpoint from wherever environment it is compiled, and stays that way. Say it is built in staging, then inherits the endpoint for staging, and if used in production, the endpoint will stay as staging. Local will stay local, etc. You get the idea.
>
>	Also, usfs-components does not currenty allow for using environment variables. So that will be changed in a new PR shortly.




*Temporary Solution:* 

> 'Frankenstein' builds; basically have a script download each repo from github, then build `usfs-components` and copy the output into `officer-form-chapters` then build, then copy that output into `officer-complaint-form`.
>
> I was able to run the latest components this way. After I finish testing making up for lost time, I will try to automate the process through a script.  
>
> The output is choppy, does not feel right, but at least allows me to see the latest code and run it.

**Comment by sergiogcx** [comment]
I would appreciate any new ideas or additional insight @amenity @briaguya @easherma @Niicck 

If you are ok with it, we could use a simple build script for now. I will get it done later tomorrow.

**Comment by sergiogcx** [comment]
TL;DR : I will be able to finish this issue and the form, to do so I just had to come up with a patchy hack to see the latest updates. @amenity @briaguya

**Comment by amenity** [comment]
@sergiogcx - @briaguya and I spoke about this and agree that patchy hack is a-ok for the time being.

**Comment by sergiogcx** [comment]
Ok, the form finally has a full cycle of deployment:

Fill out form (yes)
Submit form (yes)
API gets correct data (yes)
S3 gets the correct files (yes)
Data goes to the correct places (yes, development places, namely two emails one for the user and another for the receiving party)
The files are associated with the correct records (yes)
Confirmation number given to user matches all the data in all the places (yes)

Feel free to test it yourself:

English:
https://dxun9n0idyi09.cloudfront.net/police-oversight/complaint/share-evidence

Spanish (This is just the route issue, the form is not yet translated):
https://dxun9n0idyi09.cloudfront.net/fiscalizacion-de-policia/queja/
